src/main/java/slimeknights/tconstruct/TConstruct.java

```diff
@@ -142,4 +170,37 @@ public Container createMenu(int p_createMenu_1_, PlayerInventory p_createMenu_2_
       });
     event.getCommandDispatcher().register(executes);
   }
+
+  @SubscribeEvent // TODO: Remove after a while, maybe at release. Doesn't work right now?
+  public void missingItemMappings(RegistryEvent.MissingMappings<Item> event) {
+    for (RegistryEvent.MissingMappings.Mapping<Item> entry : event.getAllMappings()) {
+      for (SlimeBlock.SlimeType slime : SlimeBlock.SlimeType.values()) {
+        // Remap slime_sling_$slime
+        if (entry.key.getNamespace().equals(TConstruct.modID) && entry.key.getPath().equals("slime_sling_" + slime.getName())) {
```

Comment:
Add a single mod ID filter at the top, can save the whole effort of the loop if its not our mod